standard.mss

```diff
@@ -288,6 +288,12 @@
   [zoom>=10]["railway"="proposed"]["proposed_railway"="light_rail"]["service"=null],
   [zoom>=11]["railway"="construction"]["construction_railway"="tram"]["service"=null],
   [zoom>=11]["railway"="proposed"]["proposed_railway"="tram"]["service"=null],
+  [zoom>=13]["railway"="construction"]["construction_railway"="subway"]["service"!=null],
+  [zoom>=13]["railway"="proposed"]["proposed_railway"="subway"]["service"!=null],
+  [zoom>=13]["railway"="construction"]["construction_railway"="light_rail"]["service"!=null],
+  [zoom>=13]["railway"="proposed"]["proposed_railway"="light_rail"]["service"!=null],
+  [zoom>=13]["railway"="construction"]["construction_railway"="tram"]["service"!=null],
+  [zoom>=13]["railway"="proposed"]["proposed_railway"="tram"]["service"!=null],
   [zoom>=13]["railway"="construction"]["construction_railway"="narrow_gauge"],
```

Comment:
I'm just wondering if we should use the selection without the ```service``` clause for all z13 elements like it already is for narrow_gauge.

Reply:
We render service tracks of subways, light rails and trams on higher zoom levels only. It makes sense to me to apply this rule to tracks under construction as well.

Reply:
Fine with me, I just noticed not all tracks were treated the same.